tests/units/units.cpp

```diff
@@ -1138,8 +1240,8 @@ TEST(Units, complicatedMultiplicationFactorUnits)
     EXPECT_EQ(1.0, libcellml::Units::scalingFactor(u1, u2));
     EXPECT_EQ(1.0, libcellml::Units::scalingFactor(u3, u4));
     EXPECT_EQ(1e-08, libcellml::Units::scalingFactor(incredible_pile_of_square_apples, square_apple));
-    // Incompatible units but return a scaling factor.
-    EXPECT_EQ(1e-07, libcellml::Units::scalingFactor(incredible_pile_of_square_apples, bunch_of_bananas));
+    // Incompatible units but return a scaling factor of 0
```

Comment:
// Incompatible units so we return a scaling factor of 0.0.

Reply:
Done.